### PR TITLE
lrt peg checker for curve and balancer, wip

### PR DESCRIPTION
**Pull request by tapired**

### CODE REVIEW

File: lrt-pegs/balancer/main.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+from web3 import Web3
+from dotenv import load_dotenv
+import os, json, requests
+
+load_dotenv()
+
+peg_threshold = 0.05 # 5%
```
Comment (spalen0) [inline lrt-pegs/balancer/main.py:7]: Is this used for something?

File: lrt-pegs/balancer/main.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+provider_url = os.getenv("PROVIDER_URL_MAINNET")
+w3 = Web3(Web3.HTTPProvider(provider_url))
+w3_mainnet = Web3(Web3.HTTPProvider(provider_url))
+
+ASSET_BONDS_EXCEEDED = "GYR#357" # https://github.com/gyrostable/gyro-pools/blob/24060707809123e1ffd222eba99a5694e4b074c7/tests/geclp/util.py#L419
+
+PEG_THRESHOLD = 1
```
Comment (spalen0) [inline lrt-pegs/balancer/main.py:14]: Why value 1?

File: lrt-pegs/balancer/main.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    
+    total = 0
+    start_index = 1 if is_nested else 0
+
+    for i in range(start_index, len(balances)):
+        total += balances[i]
+
+    percentage = (balances[idx_lrt] / total) * 100
+    if percentage > PEG_THRESHOLD:
+        message = f"🚨 Balancer Alert! {pool_name} ratio is {percentage:.2f}% 🚀 Exceeds threshold! ⚠️"
```
Comment (spalen0) [inline lrt-pegs/balancer/main.py:54]:
```suggestion
        message = f"🚨 Balancer Alert! {pool_name} ratio is {percentage:.2f}% 🚀 "
```

Remove: "Exceeds threshold! ⚠️" it is clear there is a problem if you received tg message.

**Review by spalen0 — COMMENTED** [review]
What actions do we plan to take after the peg exceeds the threshold?

Think about using disabled notifications for small depeg like here: https://github.com/tapired/monitoring-scripts-py/blob/ad9820ca1b99427ceefde61706f4342631ce0e72/compound/main.py#L69

We don't want to have the same problem as with Matic depeg to get messages every hour but without taking any action.

File: lrt-pegs/balancer/main.py (hunk 1 above)
Comment (tapired) [reply lrt-pegs/balancer/main.py:7]: yep it will be somethignlike 70% in prod

File: lrt-pegs/balancer/main.py (hunk 2 above)
Comment (tapired) [reply lrt-pegs/balancer/main.py:14]: test

**Review by spalen0 — APPROVED** [review]

**Pull request merged**